go.mod

```diff
@@ -52,3 +52,7 @@ replace (
 	github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
 	golang.org/x/text => golang.org/x/text v0.3.3 // Required to fix CVE-2020-14040
 )
+
+// This replaced version includes controller-runtime predicate utilities necessary for v1.0.0 that are still in master.
+// Remove this and require the next minor/patch version of controller-runtime (>v0.6.1) when released.
+replace sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.6.1-0.20200724132623-e50c7b819263
```

Comment:
Can we just require this directly?

Reply:
No, the module gets reverted to v0.6.1 because of operator-lib I think.

Reply:
Hmm that's weird and annoying. You would think that a more recent commit and one that contains the references we have to libraries in that commit would win. :man_shrugging: